### ISSUE TITLE
Recursive python symlinks in virtualenv

### ISSUE THREAD

**Issue by omry** [theacodes/nox#216]
versions:
nox==2019.5.30
virtualenv==16.6.2

Trying to debug, I wanted to enter the virtualenv:
`$ virtualenv .nox/session-XYZ`

This resulted in 
OSError: [Errno 40] Too many levels of symbolic links: 'ENV_DIR/bin/python'

Looking inside, looks like python -> python3.6 and python3.6 -> python.
```

$ ls-l .nox/session-XYZ/bin | grep python
lrwxrwxrwx 1 omry omry    9 Jul 17 15:32 python -> python3.6
lrwxrwxrwx 1 omry omry    6 Jul 17 15:37 python3 -> python
lrwxrwxrwx 1 omry omry    6 Jul 17 15:37 python3.6 -> python
```

**Comment by schuderer** [comment]
Fellow user here. I tried to reproduce this, but couldn't. Is this behaviour reproducible when deleting and having .nox re-create the session virtualenvs?

I searched for the error and found nothing definitive, only some tentative indication that creating a virtualenv sometimes seems to create these recursive symlinks, possibly when it runs several times and/or with nonstandard interpreter paths (https://stackoverflow.com/questions/43853672/oserror-errno-40-too-many-levels-of-symbolic-links-home-ahmed-virtualenvs, /pypa/virtualenv/issues/1181) and/or spelling variations like upper-case or underscores vs hyphens (/pypa/virtualenv/issues/1006). Strange, and not sure how this happens. 🤷‍♂ 

Did deleting the session directory (or the whole .nox directory) fix it, or did the problem resurface again?
